ios/Podfile.lock

```diff
@@ -4274,7 +4285,7 @@ SPEC CHECKSUMS:
   FirebaseCoreDiagnostics: 3d36e05da74cb8b7ce30e6594a8f201b982c725c
   FirebaseInstallations: a58d4f72ec5861840b84df489f2668d970df558a
   fmt: ff9d55029c625d3757ed641535fd4a75fedc7ce9
-  glog: 85ecdd10ee8d8ec362ef519a6a45ff9aa27b2e85
+  glog: 73c2498ac6884b13ede40eda8228cb1eee9d9d62
```

Comment:
The hash for `glog` and `DoubleConversion` shouldn't change — could you try to remove `ios/Pods` and run `pod install` again?